jekyll/_cci2/openid-connect-tokens.adoc

```diff
@@ -174,18 +176,40 @@ You can take advantage of the format of the claims in CircleCI's <<format-of-the
 
 To do this, edit your IAM role's trust policy so that only an OIDC token from your chosen project can assume that role. The trust policy determines under what conditions the role can be assumed.
```

Comment:
What do you think about adding another heading level here, so like `===== Restrict by project` and `===== Restrict by branch` or something?

Reply:
Also we're using different methods of indicating items the reader needs to substitute in code snippets on this page.  Would be good to change to lowercase and within `< >` for all on this page. Some instances of this are outside the scope of your changes here actually so I will make another ticket to avoid holding you up :-D.